app/views/visitors/index.html.erb

```diff
@@ -1,24 +1,86 @@
-    </aside>
-    <article class="col">
-      <h1>Rails Template</h1>
-      <p>Welcome to dxw's template. This application contains our latest defaults for use when starting a new repository to get going as quickly as possible.</p>
-      <p>On first use please refer to the template <%= link_to "README", "https://github.com/dxw/dfsseta-apply-for-landing-ruby", target: :blank %> for further instructions.
-      <p>This home page should be removed during initial set up.</p>
+<main class="govuk-main-wrapper">
+  <section class="container">
+    <div class="row">
+      <article class="col">
+        <h1 class="govuk-heading-xl">Apply for landing</h1>
 
-      <h2>Helpful links</h2>
-      <ul>
-        <li>
-          <%= link_to "Architecture decision records", "https://github.com/dxw/dfsseta-apply-for-landing-ruby/tree/develop/doc/architecture/decisions" %>
-        </li>
-        <li>
-          <%= link_to "dxw RFCs", "https://github.com/dxw/tech-team-rfcs/" %>
-        </li>
-      </ul>
-    </article>
-  </div>
-</section>
+        <p class="govuk-body">Welcome to the service</p>
+        <p class="govuk-body">Refer to our <%= link_to "README", "https://github.com/dxw/dfsseta-apply-for-landing-ruby", class: "govuk-link", target: :blank %> for further instructions.
+
+        <h2 class="govuk-heading-l">GOV UK Design System "accordion" test</h2>
+          <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+            <div class="govuk-accordion__section">
+              <div class="govuk-accordion__section-header">
+                <h2 class="govuk-accordion__section-heading">
+                  <span class="govuk-accordion__section-button" id="accordion-default-heading-1">
+                    Overview
+                  </span>
+                </h2>
+              </div>
+              <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
+                <p class='govuk-body'>
+                  The DfSSETA Apply for Landing application supports a fictional Apply for Landing service supplied by the (equally fictional) Department for Space Strategy and Extra-Terrestrial Affairs.
+
+                  The non-fictional purpose of the application is to serve as a model application that provides a template for how services are designed and developed at dxw. The intention is to guide developers new to the organisation in the dxw approach to application development; to aid developers in honing their cross-language skills; to demonstrate the dxw approach to interested external parties; and to serve as a basis for experimentation and trying out new technologies and ideas in a principled and systematic way.
+
+                  The fictional purpose is to allow pilots of interplanetary spacecraft to obtain landing permits for a variety of destinations (namely, moons and planets) within Earth's solar system.
```

Comment:
Is this temporary text? If not, shall we put this in the translations?

Reply:
This accordion is a temporary demonstration that the GOV.UK Design System (JS and CSS) is working. It's removed in the next PR where we implement the "start page".

Regarding translations, yes we ought to use them! Maybe the [start page PR](https://github.com/dxw/dfsseta-apply-for-landing-ruby/pull/16) ought to finish with a refactoring commit in which we extract all copy into translations?